charts/portworx/templates/storage-cluster.yaml

```diff
@@ -473,14 +473,15 @@ spec:
     {{- end }}
   {{- end }}
 
+  {{- $csiEnabled := eq .Values.csi.enabled true }}
   {{- if or (eq .Values.csi.enabled true) (eq .Values.csi.enabled false) }}
   csi:
     enabled: {{ .Values.csi.enabled }}
-    {{- if eq .Values.csi.topology.enabled true}}
+    {{- if and ($csiEnabled) (eq .Values.csi.topology.enabled true) }}
```

Comment:
can you move the $csiEnabled to above block as nested if

Reply:
Done